tools/sprite_stuff.py

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/python3
```

Comment:
I suggest we remove this from this PR and instead push to a different branch `playersprites`